sh/src/builtin/read.rs

```diff
@@ -13,10 +13,12 @@ use crate::shell::opened_files::{OpenedFile, OpenedFiles, STDIN_FILENO};
 use crate::shell::Shell;
 use crate::wordexp::expanded_word::ExpandedWord;
 use crate::wordexp::split_fields;
-use atty::Stream;
 use nix::errno::Errno;
-use std::os::fd::{AsRawFd, RawFd};
-use std::time::Duration;
+use std::{io::stdin, time::Duration};
+use std::{
+    io::IsTerminal,
+    os::fd::{AsRawFd, RawFd},
+};
```

Comment:
This was cargo fmt - I didn't review it. Perhaps consider going with the same decision as the rust standard library does and configuring a rustfmt.toml to group imports by module? This tends to cut down a large number of lines and makes diffs simpler generally.

Reply:
https://github.com/rustcoreutils/posixutils-rs/pull/400